<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addPolygon</class>
 <widget class="QDialog" name="addPolygon">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>488</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>450</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="title">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Polygon Creator&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="shapeDimensions">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>351</width>
     <height>191</height>
    </rect>
   </property>
   <property name="title">
    <string>Shape Dimensions</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>328</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="point">
       <property name="text">
        <string>Point 1 (x1,y1)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="x1_2"/>
     </item>
     <item>
      <widget class="QLineEdit" name="y1_2"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>328</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="point_2">
       <property name="text">
        <string>Point 2 (x2,y2)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="x2_2"/>
     </item>
     <item>
      <widget class="QLineEdit" name="y2_2"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>328</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_13">
     <item>
      <widget class="QLabel" name="point_6">
       <property name="text">
        <string>Point 3 (x3,y3)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="x3_2"/>
     </item>
     <item>
      <widget class="QLineEdit" name="y3_2"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>328</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_14">
     <item>
      <widget class="QLabel" name="point_7">
       <property name="text">
        <string>Point 4 (x4,y4)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="x4_2"/>
     </item>
     <item>
      <widget class="QLineEdit" name="y4_2"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>328</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_15">
     <item>
      <widget class="QLabel" name="point_8">
       <property name="text">
        <string>Point 5 (x5,y5)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="x5_2"/>
     </item>
     <item>
      <widget class="QLineEdit" name="y5_2"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="penProperties">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>351</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>Pen Properties</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>136</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="penColor">
       <property name="text">
        <string>Pen Color</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="penColorBox">
       <item>
        <property name="text">
         <string>---</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>White</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Black</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Red</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Green</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Blue</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Cyan</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Magenta</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Yellow</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Gray</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>102</width>
      <height>27</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="penWidth">
       <property name="text">
        <string>Pen Width</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="penWidthBox">
       <property name="maximum">
        <number>20</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>171</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="penStyle">
       <property name="text">
        <string>Pen Style</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="penStyleBox">
       <item>
        <property name="text">
         <string>---</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>NoPen</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SolidLine</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>DashLine</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>DotLine</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>DashDotLine</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>DashDotDotLine</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>166</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="penCapStyle">
       <property name="text">
        <string>Pen Cap Style</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox">
       <item>
        <property name="text">
         <string>---</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>FlatCap</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SquareCap</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>RoundCap</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>165</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QLabel" name="penJoinStyle">
       <property name="text">
        <string>Pen Join Style</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_2">
       <item>
        <property name="text">
         <string>---</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>MiterJoin</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>BevelJoin</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>RoundJoin</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="brushProperties">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>400</y>
     <width>271</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Brush Properties</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>146</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QLabel" name="brushColor">
       <property name="text">
        <string>Brush Color</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_3">
       <item>
        <property name="text">
         <string>---</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>White</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Black</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Red</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Green</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Blue</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Cyan</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Magenta</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Yellow</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Gray</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>159</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <widget class="QLabel" name="brushStyle">
       <property name="text">
        <string>Brush Style</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_4">
       <item>
        <property name="text">
         <string>---</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SolidPattern</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>HorPattern</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>VerPattern</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>NoBrush</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>addPolygon</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>addPolygon</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
